python/mosaic/api/functions.py

```diff
@@ -423,26 +422,7 @@ def st_centroid2D(geom: ColumnOrName) -> Column:
 
     """
     return config.mosaic_context.invoke_function(
-        "st_centroid2D", pyspark_to_java_column(geom)
-    )
-
-
-def st_centroid3D(geom: ColumnOrName) -> Column:
```

Comment:
@milos-colic is the removal of the st_centroid3D function tied to removing Esri JAR or something else?

Reply:
We dont have 3D support yet - this confuses the story, and wont work actually in JTS nor ESRI correctly, so better cut it out before we have a proper 3D support.